tutorials/hyperspectral/RasterStack-RGB-Images-in-R-Using-HSI.Rmd

```diff
@@ -74,16 +77,12 @@ on the ground.
 
 <figure >
     <a href="{{ site.baseurl }}/images/hyperspectral/RGBImage_2.png"><img src="{{ site.baseurl }}/images/hyperspectral/RGBImage_2.png"></a>
-<figcaption> A portion of the SJER field site using red, green and blue (bands 58,34,19).</figcaption>
+<figcaption> A portion of the SJER field site using red, green and blue (example dataset bands 14,9,4; bands 58,34,19 in the full NEON dataset).</figcaption>
 </figure>
 
-<figure class="half">
+<figure>
     <a href="{{ site.baseurl }}/images/hyperspectral/NIR_G_B.png"><img src="{{ site.baseurl }}/images/hyperspectral/NIR_G_B.png"></a>
-    <a href="{{ site.baseurl }}/images/hyperspectral/falseColor.png"><img src="{{ site.baseurl }}/images/hyperspectral/falseColor.png"></a>
-    
-    <figcaption>Here is the same section of SJER but with other bands highlighted. 
-    LEFT: colored infrared – near infrared, green and blue (bands 90, 34, 19). 
-    RIGHT: A false color image.</figcaption>
+<figcaption> Here is the same section of SJER but with other bands highlighted to create a colored infrared image – near infrared, green and blue (example dataset bands 22, 9, 4; bands 90, 34, 19 in the full NEON dataset).</figcaption>
 </figure>
 
 <div id="ds-dataTip" markdown="1">
```

Comment:
I can't add comments the the part that isn't edited it appears.  Line 92- Missing ) after The tool requires Flash player.

Reply:
Good catch - corrected.